### ISSUE TITLE
meta set tests.coverage does not work with doubles

### ISSUE THREAD

**Issue by dragon512** [screwdriver-cd/screwdriver#1559]
in SD job running
meta set tests.coverage 33.3
or 44.0002

does a no-op

I would except a value as 
33.3% or 44% to show up

**Comment by reid** [comment]
In screwdriver-cd/ui#413, data validation was added which requires `tests.coverage` to be an integer. Otherwise, it will not be shown, and the default coverage data from tools like Sonar will be displayed instead.

I believe this was done by mistake. See: https://github.com/screwdriver-cd/ui/commit/7d0cd630b175e10c60ceff45d7b811223cc68e20#diff-2a3a32d07ab45174e1b110a7c3a6f014R84

We ran into this bug today. The good news is the meta value should appear in the UI once the UI bug is fixed.
